conceptnet5/vectors/transforms.py

```diff
@@ -1,9 +1,12 @@
-import pandas as pd
+import msgpack
 import numpy as np
+import pandas as pd
+from annoy import AnnoyIndex
```

Comment:
I think that `'annoy'` needs to be added to the "vectors" requirements in `setup.py`.